src/scripts/script.js

```diff
@@ -0,0 +1,209 @@
+const selectedRates = ["BTC", "ETC", "ETH", "LTC", "XRP"];
+
+function returnImgName(abr) {
+  switch (abr) {
+    case "BTC":
+      return "btc";
+
+    case "ETC":
+      return "etc";
+
+    case "ETH":
+      return "eth";
+
+    case "LTC":
+      return "ltc";
+
+    case "XRP":
+      return "xrp";
+
+    default:
+      return "btc";
+  }
+}
+
+function returnFullName(abr) {
+  switch (abr) {
+    case "BTC":
+      return "Bitcoin";
+
+    case "ETC":
+      return "Etherium Classic";
+
+    case "ETH":
+      return "Etherium";
+
+    case "LTC":
+      return "Litecoin";
+
+    case "XRP":
+      return "Ripple";
+
+    default:
+      return "Bitcoin";
+  }
+}
+
+function returnCurrencySymbol(curr) {
+  switch (curr) {
+    case "BTC":
+      return "Ƀ";
+
+    case "EUR":
+      return "€";
+
+    case "GBP":
+      return "£";
+
+    case "USD":
+      return "$";
+
+    default:
+      return "£";
+  }
+}
+
+// document.addEventListener("DOMContentLoaded", () => {
```

Comment:
Remove commented out code.
This code is in version control. If you need to refer to it again in the future you can just checkout an earlier commit. Keeping commented out code in the branch is bad practice.